resources/src/slack/add_reaction.rs

```diff
@@ -3,7 +3,7 @@ use slack_morphism::prelude::*;
 use crate::slack::client::build_token;
 
 // This function will add the specified reaction to a message identified by its timestamp.
-pub async fn run(
+pub async fn add_reaction(
```

Comment:
Is this commit on top of a super old version of the code? This is missing all my QoL changes. 

https://github.com/ASU-CodeDevils/devil_bot_rust/pull/48/files